hw/dv/data/common_sim_cfg.hjson

```diff
@@ -23,12 +23,17 @@
   sw_root_dir:        "{proj_root}/sw"
 
   // pass and fail patterns
-  pass_patterns:    ["^TEST PASSED (UVM_)?CHECKS$"]
-  fail_patterns:    ["^UVM_ERROR\\s[^:].*$",
-                     "^UVM_FATAL\\s[^:].*$",
-                     "^Assert failed: ",
-                     "^\\s*Offending '.*'",
-                     "^TEST FAILED (UVM_)?CHECKS$"]
+  compile_pass_patterns: []
+  compile_fail_patterns: ["^Error-.*$",
+                          "\\*E.*$"]
+  sim_pass_patterns: ["^TEST PASSED (UVM_)?CHECKS$"]
+  sim_fail_patterns: ["^UVM_ERROR\\s[^:].*$",
+                      "^UVM_FATAL\\s[^:].*$",
+                      "^Assert failed: ",
+                      "^\\s*Offending '.*'",
+                      "^TEST FAILED (UVM_)?CHECKS$",
+                      "^Error-.*$", // Null pointer error
+                      "\\*E.*$"]
```

Comment:
Same comment as above - split these for VCS and Xcelium. The tool aggregates lists with the same names encountered in different files, so it should work.

Reply:
Done